tests/unit/hive_metastore/test_table_migrate.py

```diff
@@ -1349,3 +1349,64 @@ def test_revert_migrated_tables_failed(caplog):
     table_migrate = get_table_migrator(backend)
     table_migrate.revert_migrated_tables(schema="test_schema1")
     assert "Failed to revert table hive_metastore.test_schema1.test_table1: error" in caplog.text
+
+
+def test_refresh_migration_status_published_remained_tables(caplog):
+    errors = {}
+    rows = {}
+    backend = MockBackend(fails_on_first=errors, rows=rows)
```

Comment:
These are arguments are optional, only specify them if you use them

```suggestion
    backend = MockBackend()
```